src/components/dashboard/containers/DashboardStats/BasicStats.js

```diff
@@ -208,6 +220,30 @@ export function BasicStats({ side }) {
 		execute(updateAnalyticsPeriodPrevNextLive({ live: true }))
 	}, [side])
 
+	useEffect(() => {
+		loop && loop.startLoop()
+		return () => {
+			loop && loop.stop()
+		}
+	}, [loop])
```

Comment:
need to add loop stop on "unmount" use `useEffect` with `[]` for params, This way throttled analytics update may not be needed